client/a8c-for-agencies/style.scss

```diff
@@ -600,6 +600,10 @@ html {
 		padding-inline: 0;
 	}
 
+	.hosting-dashboard-layout__top-wrapper > * {
+		max-width: none;
```

Comment:
For a full-width table page like Referrals, Commissions etc we want the layout top to take full width irrespective of the screen size